controller/instance_manager_controller.go

```diff
@@ -547,15 +544,26 @@ func (imc *InstanceManagerController) syncLogSettingsToInstanceManagerPod(im *lo
 		}
 
 		switch settingName {
-		case types.SettingNameV2DataEngineLogLevel:
-			err = client.LogSetLevel(longhorn.DataEngineTypeV2, "spdk_tgt", setting.Value)
+		case types.SettingNameLogLevel:
+			// We use this to set the instance-manager log level, for either engine type.
+			err = client.LogSetLevel(longhorn.DataEngineTypeV1, "", setting.Value)
```

Comment:
This is quite confusing. Can we replace `longhorn.DataEngineTypeV1` with `""`?

Reply:
Yes, that should work.  `getDataEngine` in longhorn-instance-manager/pkg/client should translate `""` that way.

Reply:
Tested and re-pushed.